pkg/meta/index.go

```diff
@@ -17,6 +17,7 @@ package meta
 
 type Index struct {
 	Name        string                 `json:"name"`
+	Aliases     []string               `json:"aliases"`
```

Comment:
Instead of storing aliases to the index, will storing index to alias make more sense? Let's discuss.

Reply:
Yeah we can discuss this.